### PR TITLE
Fix linting

### PR DESCRIPTION
**Pull request by Alfred-Mountfield**
A lint failure managed to squeak through onto main

### CODE REVIEW

File: site/src/pages/workshop/workshop.md [hunk 1]
```diff
@@ -375,9 +375,9 @@ export const App: BlockComponent<BlockEntityProperties> = ({
 }) => {
   ...
   // find only the entities in linkedEntities which appear as the destination of one of the projectLinks
-  const 
-  
-  
+  const
```
Comment (Alfred-Mountfield) [inline site/src/pages/workshop/workshop.md:378]: This looks broken anyway? 🤔

Comment (kachkaev) [reply site/src/pages/workshop/workshop.md:378]: @Alfred-Mountfield I fixed syntax inside ` ``` ` in https://github.com/blockprotocol/blockprotocol/commit/4d901b9a57f8a462406e781704f641dccf50e672. Feel free to cherry-pick.

Comment (Alfred-Mountfield) [reply site/src/pages/workshop/workshop.md]: Cherry-picked and pushed, thanks

**Review by TimDiekmann — APPROVED** [review]
Looks good 🙂

**Pull request merged**